docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
   postgres:
     image: postgres:9.6
     ports:
-      - "5432"
+      - "5432:5432"
```

Comment:
expose the DB to the host, makes it easier to connect with external tools.

Reply:
I would go for that change. @Pomax is it a problem?